src/classes/BDI_DataImportService.cls

```diff
@@ -595,7 +617,10 @@ global with sharing class BDI_DataImportService {
     * @return void
     */
     private void customPostProcess() {
-        BDI_IPostProcess iPostProcess = (BDI_IPostProcess)instantiateClassForInterface('BDI_IPostProcess', diSettings.Post_Process_Implementing_Class__c);
+        BDI_IPostProcess iPostProcess =
+            (BDI_IPostProcess)instantiateClassForInterface(
```

Comment:
Do we add a space after casting records, ie `(BDI_IPostProcess) instantiateClassForInterface()`?

Reply:
I typically don't, but I don't have a strong opinion either way.